pkg/adapter/config/cacheability.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import "time"
+
+// CacheabilityInfo represents information about when to refresh the cache.
+type CacheabilityInfo struct {
+	ValidDuration time.Duration
+	ValidUseCount int64
```

Comment:
We need to define/document the "don't care" values for each too. I expect the most common case is that adapters don't care about `ValidUseCount` and only want to set a TTL; how do they do that?